app/recipeApp.js

```diff
@@ -0,0 +1,339 @@
+let recipeObject
+
+let recipes = []; 
```

Comment:
having a global variable in this case is not needed. You can return the data from getData and then pass it as a parameter to `showAllRecipes`